app/src/main/java/net/squanchy/service/firestore/FirestoreMappers.kt

```diff
@@ -0,0 +1,66 @@
+package net.squanchy.service.firestore
+
+import net.squanchy.eventdetails.domain.view.ExperienceLevel
+import net.squanchy.schedule.domain.view.Event
+import net.squanchy.schedule.domain.view.Place
+import net.squanchy.schedule.domain.view.Track
+import net.squanchy.service.firebase.model.FirebaseFavorites
+import net.squanchy.service.firestore.model.conferenceinfo.FirestoreVenue
+import net.squanchy.service.firestore.model.schedule.FirestoreEvent
+import net.squanchy.service.firestore.model.schedule.FirestorePlace
+import net.squanchy.service.firestore.model.schedule.FirestoreSpeaker
+import net.squanchy.service.firestore.model.schedule.FirestoreTrack
+import net.squanchy.speaker.domain.view.Speaker
+import net.squanchy.support.lang.Checksum
+import net.squanchy.support.lang.optional
+import net.squanchy.venue.domain.view.Venue
+import org.joda.time.DateTimeZone
+import org.joda.time.LocalDateTime
+
+fun FirestorePlace.toPlace(): Place = Place.create(id, name, floor.optional())
```

Comment:
These should be very easy to test! Can we have tests plz?

Reply:
Will do a PR on top of this one for testing, working on it